File: samples/server/petstore/jaxrs-cxf-non-spring-app/src/main/java/io/swagger/api/impl/UserApiServiceImpl.java
package io.swagger.api.impl;

import io.swagger.api.*;
import java.util.List;
import io.swagger.model.User;

import java.io.InputStream;
import java.io.OutputStream;
import java.util.List;
import java.util.Map;
import javax.ws.rs.*;
import javax.ws.rs.core.Response;
import org.apache.cxf.jaxrs.model.wadl.Description;
import org.apache.cxf.jaxrs.model.wadl.DocTarget;

import org.apache.cxf.jaxrs.ext.multipart.*;

import io.swagger.annotations.Api;

public class UserApiServiceImpl implements UserApi {
    public void createUser(User body) {
        // TODO: Implement...
        
        
    }
    
    public void createUsersWithArrayInput(List<User> body) {
        // TODO: Implement...
        
        
    }
    
    public void createUsersWithListInput(List<User> body) {
        // TODO: Implement...
        
        
    }
    
    public void deleteUser(String username) {
        // TODO: Implement...
        
        
    }
    
    public User getUserByName(String username) {
        // TODO: Implement...
        
        return null;
    }
    
    public String loginUser(String username, String password) {
        // TODO: Implement...
        
        return null;
    }
    
    public void logoutUser() {
        // TODO: Implement...
        
        
    }
    
    public void updateUser(String username, User body) {
        // TODO: Implement...
        
        
    }
    
}

File: samples/server/petstore/kotlin-springboot-responseentity/src/main/kotlin/org/openapitools/api/UserApiServiceImpl.kt
package org.openapitools.api

import org.openapitools.model.User
import org.springframework.stereotype.Service
import org.springframework.http.ResponseEntity


@Service
class UserApiServiceImpl : UserApiService {

    override fun createUser(body: User): ResponseEntity<Unit> {
        TODO("Implement me")
    }

    override fun createUsersWithArrayInput(body: kotlin.collections.List<User>): ResponseEntity<Unit> {
        TODO("Implement me")
    }

    override fun createUsersWithListInput(body: kotlin.collections.List<User>): ResponseEntity<Unit> {
        TODO("Implement me")
    }

    override fun deleteUser(username: kotlin.String): ResponseEntity<Unit> {
        TODO("Implement me")
    }

    override fun getUserByName(username: kotlin.String): ResponseEntity<User> {
        TODO("Implement me")
    }

    override fun loginUser(username: kotlin.String, password: kotlin.String): ResponseEntity<kotlin.String> {
        TODO("Implement me")
    }

    override fun logoutUser(): ResponseEntity<Unit> {
        TODO("Implement me")
    }

    override fun updateUser(username: kotlin.String, body: User): ResponseEntity<Unit> {
        TODO("Implement me")
    }
}
